deploy/ray-operator/config/crd/bases/ray.io_rayclusters.yaml

```diff
@@ -671,6 +671,9 @@ spec:
                       be preserved when modifying objects. Optional. Usage refers
                       to https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
                     type: object
+                  args:
+                    description: Args to start ray
```

Comment:
Same question like above, how to distinguish `args` from `command`?

Reply:
Removed as not used.